api/models/comment-models.js

```diff
@@ -0,0 +1,14 @@
+const connection = require("../../db/connection");
+const { checkExists } = require("../../api/utility-functions/api-utils");
+
+exports.removeCommentById = (commentid) => {
+  const promiseCatcher = [checkExists("comments", "comment_id", commentid)];
+  const valueCatcher = [commentid];
```

Comment:
I'd argue that this valueCatcher variable is less clear than just passing in `[commentid]` in directly